src/product/types.ts

```diff
@@ -0,0 +1,20 @@
+import { GotPromise } from 'got';
+import { ApiInstance, FilterParams } from '../_generic/types';
+
+export type Product = {
+  readonly id?: string;
+  readonly createdAt?: Date;
+  readonly cadastradoAutomaticamente?: boolean;
+  readonly nome?: string;
+  readonly idExterno?: string;
+  readonly diasGarantia?: number;
+  readonly valorTotal?: number;
+  readonly aliquotaIss?: any;
+  readonly discriminacaoServico?: any;
+  readonly tags?: string;
```

Comment:
This field could be an array. Then we would need to map the object before calling createProduct. I'm refactoring this thing for the sales endpoint and I'll create a PR that you can refer to in order to know what I'm talking about.

Reply:
Here's the PR: https://github.com/andremw/enotas/pull/20

Reply:
Is "tags" field the same of Sale's route?

Reply:
I don't think it's the same, but the idea is. Instead of separating the tags by a semicolon, we can provide the user with a better API to handle the tags, instead of having to split it by himself.